.gitlab.mk

```diff
@@ -1,9 +1,14 @@
 TNT_VER=$(shell cat versions/${OS}_${DIST}_${VER})
+ROCKS_INSTALLER?='tarantoolctl rocks'
 
 build:
-	docker build --no-cache --network=host --build-arg TNT_VER=${TNT_VER} \
+	[ "${DOCKERFILE_NAME_SUFFIX}" != "" ] && \
+		DOCKERFILE_SUFFIX=_${DOCKERFILE_NAME_SUFFIX} ; \
```

Comment:
It looks like the changes in the work with the DOCKERFILE_SUFFIX are not related to the commit theme and can be moved to a separate commit.

Reply:
> It looks like the changes in the work with the DOCKERFILE_SUFFIX are not related to the commit theme and can be moved to a separate commit.

Right, moved to "Enable manual build with makefile"  commit.